src/main/sensors/boardalignment.c

```diff
@@ -73,7 +73,7 @@ static void alignBoard(float *vec)
     vec[Z] = (boardRotation[0][Z] * x + boardRotation[1][Z] * y + boardRotation[2][Z] * z);
 }
 
-FAST_CODE void alignSensors(float *dest, uint8_t rotation)
+void alignSensors(float *dest, uint8_t rotation)
 {
     const float x = dest[X];
```

Comment:
It doesn't look like a bugfix.

Reply:
Please look at #6183.

Reply:
It doesn't provide any sort of explanation.
`alignSensors` is used for gyro, mag and acc. Other targets have no issues with it, this function should be `FAST_CODE`

Reply:
I haven't tested on all other targets but I'm sure that it's not working on the YupiF7 target with the 3.4 RC2, despite no relevant changes in the target.h file since the 3.3.3 release. The orientation was correct with all 3.3.x releases.

I have performed some debugging and I got the following results with the 3.4 RC2 : 
- before calling alignSensors(acc.accADC, acc.dev.accAlign); (line 523 of acceleration.c) in the accUpdate function, acc.dev.accAlign=2, which is in line with the target.h setting CW90_DEG
- inside de function alignSensor (line 81 of boardalignment.c), rotation = 2, which is correct but the orientation is not

If I replace the "rotation" variable in line 82 of boardalignement.c by CW90_DEG or 2 in hard in the code, the orientation is correct so I though that there should be a compilation optimisation issue. I tested without the FAST_CODE and this is working.

Reply:
It doesn't mean that removing `FAST_CODE` is the right solution.

Reply:
If you find the right solution, go for it. But This is a safety issue, thus it can't remain in the official release. 
Even if we right everywhere that first tests must be done without props, there is still users thinking that they can flash a new release and go flying straight.

Reply:
I'll study the code today and try to find the cause, we can't just disable fast code in this function for every target :(